src/code/PreRender.c

```diff
@@ -408,15 +693,15 @@ void PreRender_ApplyFilters(PreRender* this) {
     } else {
         this->unk_4D = 1;
         PreRender_ApplyAntiAliasingFilter(this);
-        func_801717F8(this);
+        PreRender_DivotFilter(this);
         this->unk_4D = 2;
     }
 }
 
-extern SlowlyMgr sSlowlyMgr;
-extern s32 D_801F6FC0;
-extern StackEntry sSlowlyStackInfo;
-extern STACK(sSlowlyStack, 0x1000);
+SlowlyMgr sSlowlyMgr;
+s32 D_801F6FC0;
```

Comment:
Based on usage this seems to indicate if Prerender has an active Slowly thread running. Perhaps something like `sSlowlyRunning`?